go/teams/loader_chain_test.go

```diff
@@ -62,9 +69,13 @@ func TestUnits(t *testing.T) {
 	}
 	files, err := ioutil.ReadDir(jsonDir)
 	require.NoError(t, err)
+	selectUnit := os.Getenv("KEYBASE_TEAM_TEST_SELECT")
 	var runLog []string
 	for _, f := range files {
 		if !f.IsDir() && strings.HasSuffix(f.Name(), ".json") {
+			if len(selectUnit) > 0 && f.Name() != selectUnit {
+				continue
+			}
```

Comment:
Yep, luckily it will flip this line `require.NotZero(t, runLog, "found no test units")` in the case that it matches nothing.